Dockerfile

```diff
@@ -0,0 +1,8 @@
+# syntax=docker/dockerfile:1
+   
+FROM python:3.9.6
+WORKDIR /app
+COPY . .
+RUN pip3 install -r requirements.txt
```

Comment:
You might need to have a different requrements.txt, as I don't think mod_wsgi is required in the container